src/transformers/models/m2m_100/tokenization_m2m_100.py

```diff
@@ -290,6 +290,15 @@ def prepare_seq2seq_batch(
         self.set_src_lang_special_tokens(self.src_lang)
         return super().prepare_seq2seq_batch(src_texts, tgt_texts, **kwargs)
 
+    def _build_translation_inputs(self, raw_inputs, src_lang: Optional[str], tgt_lang: Optional[str], **extra_kwargs):
+        if src_lang is None or tgt_lang is None:
```

Comment:
A one-line docstring or comment for us internally to remember where this is used would be super helpful! (putting it on this one but it applies to the others as well).

Reply:
Done